v3/wuque/creek70/creek70.json

```diff
@@ -0,0 +1,259 @@
+{
+  "name": "Creek 70",
+  "vendorId": "0x916D",
+  "productId": "0x0001",
+  "keycodes": ["qmk_lighting"],
+  "menus": [
+    {
+      "label": "Lighting",
+      "content": [
+        {
+          "label": "Rigth_Color",
+          "content": [
+            {
+              "label": "Brightness",
+              "type": "range",
+              "options": [0, 255],
+              "content": ["id_qmk_rgb_matrix_channel", 3, 1]
+            },
+            {
+              "label": "Effect",
+              "type": "dropdown",
+              "content": ["id_qmk_rgblight_effect", 3, 2],
+              "options": [
+                "All Off",
+                "Solid Color",
+                "Breathing",
+                "Rainbow Moving Chevron",
+                "Rainbow Beacon",
+                "Rainbow Pinwheels"
+              ]
```

Comment:
From your keyboard `config.h` there are only Breathing, Rainbow Mood and Static gradient. Is this correct?

Reply:
yes, different kinds of Rainbow affection is for rgb matrix.

Reply:
However you do not have RGB matrix at all? You only have RGB Light.

![image](https://user-images.githubusercontent.com/17491233/219010150-1686fbab-eefd-4dcf-a009-fbb9351d7e77.png)

You only have 
```
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
```

Reply:
I added long time ago, seill no one check. ( https://github.com/qmk/qmk_firmware/pull/19858